tox.ini

```
@@ -81,6 +81,17 @@ deps =
 commands =
     pytest --import-mode=append {posargs} src/metatensor/models/soap_bpnn/tests/
 
+[testenv:sparse-gap-tests]
+description = Run SparseGAP tests with pytest
+passenv = *
+deps =
+    pytest
+    skmatter
+    metatensor-learn
+    scipy
```

This way we test the extra requirements specified in the pyproject.toml
```suggestion

extras = sparse-gap
```